src/cat-data-service/src/service/common/objects/stake_public_key.rs

```diff
@@ -0,0 +1,12 @@
+use poem_openapi::{types::Example, NewType};
+use serde::Deserialize;
+
+#[derive(NewType, Deserialize)]
+#[oai(example = true)]
+pub(crate) struct StakePublicKey(pub String);
+
+impl Example for StakePublicKey {
+    fn example() -> Self {
+        Self("ad4b948699193634a39dd56f779a2951a24779ad52aa7916f6912b8ec4702cee".into())
```

Comment:
Hex strings must have `0x`

```suggestion
        Self("0xad4b948699193634a39dd56f779a2951a24779ad52aa7916f6912b8ec4702cee".into())
```